src/lib/dataApi.ts

```diff
@@ -209,6 +209,13 @@ export function createDataRecord(
     path = project.newNotesFolder;
   }
 
+  if (project.dataSource.kind == "tag") {
+    values = {
+      ...values,
+      ["tags"]: [project.dataSource.config.tag.replace("#", "")],
```

Comment:
Adapt to the new `Properties` feature, we can only accept tags by default.

<img width="960" alt="image" src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/06c5f67a-da90-4c1c-9eaf-8c26ce4c2516">